src/Context.js

```diff
@@ -44,3 +32,5 @@ export default class Context {
     return this.endpoints || {};
   }
 }
+
+export default new Context()
```

Comment:
According to the ES2015 specs it's not required at the end of a `export` statement. But personally I feel that it's more consistent. Though, I'd rather use a semicolon...